### PR TITLE
Update chart version

### PR DESCRIPTION
**Pull request by PaulineMauryL**
Wait for release on Lomas before merging

### CODE REVIEW

**Comment by damienbfs** [comment]
I think there have been changes to the server config that we might have to add to the chart values and values.schema (I remember at least private_db_credentials but there could be more)

File: charts/lomas/values.yaml [hunk 1]
```diff
@@ -66,8 +69,11 @@ lomas-server:
             contrib: True
             floating_point: True
             honest_but_curious: True
-
-
+        private_db_credentials:
```
Comment (damienbfs) [inline charts/lomas/values.yaml:72]: I would keep this to empty list by default, since there are no private db that use these credentials.

**Comment by damienbfs** [comment]
We also need to update the image tag in the values.schema (remove 0.3.1 and add 0.3.4).

**Review by LancelotMarti — APPROVED** [review]

**Review by damienbfs — APPROVED** [review]

**Comment by olevitt** [comment]
> Hi @olevitt , we don't see the version 0.3.4 available on Onyxia despite this being merged (it is still version 0.3.1 only in the UI). Were there any changes in what we should do for it to be visible ? or maybe we made a mistake.
> 
> Thank you very much :)

Paging @ihiverlet as I'm currently on vacation

**Comment by ihiverlet** [comment]
Hi, there was a problem while refreshing the catalogs. You should see version 0.3.4 now. However there seems to be a problem with the chart. You may need to add default values (see serviceaccount.annotations for example)

**Comment by PaulineMauryL** [comment]
> Hi, there was a problem while refreshing the catalogs. You should see version 0.3.4 now. However there seems to be a problem with the chart. You may need to add default values (see serviceaccount.annotations for example)

Thank you! We'll fix it.

**Pull request merged**